namespace/backwards-compatibilty.adoc

```diff
@@ -13,3 +13,7 @@ Build-time solutions of this nature could be achived.
 ## Runtime javaagent
 
 A runtime version of the above technique could be achived with a Java Agent. An agent has the ability to see all bytecode being loaded and to modify it before it becomes a class.
+
+## Jakarta EE container ClassLoader
+
+A corner case which cannot be handled by Runtime javaaganet is calling javax package via reflection APIs. A Jakarta EE implementation can hook ClassLoader and convert package names.
```

Comment:
Or the byte code translation can insert a name-mapping call before calling
certain ClassLoader methods.